<html class=" js flexbox flexboxlegacy canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers no-applicationcache svg inlinesvg smil svgclippaths" style="" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
		<meta charset="utf-8">

		<!-- Always force latest IE rendering engine (even in intranet) & Chrome Frame
		Remove this if you use the .htaccess -->
		<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

		<title>tRNAscan-SE Search Server</title>
		<meta name="description" content="tRNAscan-SE 2.0">
		<meta name="author" content="Todd M. Lowe and Patricia P. Chan">

		<meta name="viewport" content="width=device-width; initial-scale=1.0">

		<!-- Replace favicon.ico & apple-touch-icon.png in the root of your domain and delete these references -->
		<link rel="shortcut icon" href="http://lowelab.ucsc.edu/favicon.ico">
		<link rel="apple-touch-icon" href="http://lowelab.ucsc.edu/apple-touch-icon.png">
		<link rel="stylesheet" href="MG655623_results_files/normalize.css">
		<link rel="stylesheet" href="MG655623_results_files/foundation.css">
		<link rel="stylesheet" href="MG655623_results_files/trnascanse.css">
		<link rel="stylesheet" id="font-awesome-css" href="MG655623_results_files/font-awesome.css">
		<script src="MG655623_results_files/modernizr.js"></script>
		<script src="MG655623_results_files/jquery.js"></script>
  	<meta class="foundation-data-attribute-namespace"><meta class="foundation-mq-xxlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-large-only"><meta class="foundation-mq-large"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-small-only"><meta class="foundation-mq-small"><style></style></head>

	<body>

<!--small off canvas menu-->
<div class="off-canvas-wrap" data-offcanvas="">
	<div class="inner-wrap">

<!--top menu-->
<nav class="tab-bar show-for-small">
	<a class="left-off-canvas-toggle menu-icon ">
		<span>Home</span>
	</a>  
</nav>

<nav class="top-bar hide-for-small" data-topbar="">
	<ul class="title-area">
		<li class="name">
			<h1><a href="http://lowelab.ucsc.edu/tRNAscan-SE/index.html">Home</a></h1>
		</li>
	</ul>

	<section class="top-bar-section">
		<ul class="right">
			<li class="divider"></li>
			<li><a href="http://lowelab.ucsc.edu/tRNAscan-SE/help.html">Help</a></li>
			<li class="divider"></li>
			<li><a href="http://lowelab.ucsc.edu/tRNAscan-SE/citation.html">Citation</a></li>
			<li><a href="http://lowelab.ucsc.edu/tRNAscan-SE/contact.html">Contact Us</a></li>
		</ul>
	</section>
</nav>

<aside class="left-off-canvas-menu">

<ul class="off-canvas-list">
	<li><label class="first">tRNAscan-SE</label></li>
	<li><a href="http://lowelab.ucsc.edu/tRNAscan-SE/index.html">Home</a></li>
</ul>

</aside>


<!--content-->
<!--title-->
<header id="page-header">	  
	<div class="small-10 medium-7 large-4 columns">
		<img src="MG655623_results_files/tRNAscanSE_with_name.png">
	</div>
	<div class="small-10 medium-7 large-4 columns">
		<h5>tRNAscan-SE Results</h5>
	</div>
</header>

<div class="scroll-top-wrapper">
	<span class="scroll-top-inner">
		<i class="fa fa-2x fa-arrow-up"></i>
	</span>
</div>

<section id="genome-section">	
	<div class="row">
		<div class="small-24 medium-12 large-12 columns" style="overflow:auto">

<center>
<h5>tRNA search in progress.</h5><img src="MG655623_results_files/trna-spin-white-bg.gif"><br>
(Total length of sequence(s) = 13339
 bp)
<br><br><br><br>Please note that no hmm-filter searches are very slow.<br>Accordingly, please limit the size of sequences submitted using these search modes, <br>or download the UNIX source and run the program locally...<p><br>
Please Wait...<br>
</p></center>
<script type="text/javascript">
    var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
    document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script><script src="MG655623_results_files/ga.js" type="text/javascript"></script>
<script type="text/javascript">
try {
    var pageTracker = _gat._getTracker("UA-4200031-5");
    pageTracker._trackPageview();
} catch(err) {}
</script>



<center><br>
<h4>Search Completed</h4><br>
</center>
<h4><b>tRNAscan-SE v. 2.0</b></h4>

<i>Please cite</i>:<br><br>
<ul>
<li>Lowe, T.M. and Chan, P.P. (2016) tRNAscan-SE On-line: Search and 
Contextual Analysis of Transfer RNA Genes. Nucl. Acids Res. 44: W54-57.</li>
<li>Chan, P.P., Lin, B., and Lowe, T.M. tRNAscan-SE 2.0. (In Preparation)</li>
</ul>
tRNA secondary structures rendered by NAVIEW (c) 1988 Robert E. Bruccoleri.<br>

<hr>

<h4><b>Results</b></h4>

<b>No tRNAs found</b>



<hr>
<h5><b>Run Statistics:</b></h5>


<table width="600px">
<thead><tr><th colspan="2">Search options</th></tr></thead>
<tbody>
<tr><td width="45%">Search Mode</td><td>                       Organellar</td></tr>
<tr><td width="45%">Searching with</td><td>                    Infernal single-pass scan</td></tr>
<tr><td colspan="2">                                   Maximum sensitivity mode</td></tr>
<tr><td width="45%">Covariance model</td><td>                  models/TRNAinf-1415.cm</td></tr>
<tr><td colspan="2">tRNA covariance model search       </td></tr>
<tr><td width="45%">    cutoff score</td><td>                  15</td></tr>
<tr><td colspan="2">Pseudogene checking disabled</td></tr>
</tbody>
</table>

<table width="600px">
<thead><tr><th colspan="2">Infernal Stats</th></tr></thead>
<tbody>
<tr><td width="45%">Sequences read</td><td>           1
</td></tr>
<tr><td width="45%">Infernal-confirmed tRNAs</td><td>     0
</td></tr>
<tr><td width="45%">Bases scanned by Infernal</td><td>  13339
</td></tr>
<tr><td width="45%">% seq scanned by Infernal</td><td>  100.0 %
</td></tr>
<tr><td width="45%">Script CPU time</td><td>          0.00 s
</td></tr>
<tr><td width="45%">Infernal CPU time</td><td>            8.53 s
</td></tr>
<tr><td width="45%">Scan speed</td><td>               1563.8 bp/sec
</td></tr>
</tbody>
</table>
<table width="600px">
<thead><tr><th colspan="2">Processing time</th></tr></thead>
<tbody>
<tr><td width="45%">Started</td><td>Wed Apr 10 17:49:47 PDT 2019</td></tr>
<tr><td width="45%">Infernal analysis of tRNAs ended</td><td>Wed Apr 10 17:49:55 PDT 2019</td></tr>
<tr><td width="45%">Overall scan speed</td><td>1563.8 bp/sec</td></tr>
</tbody>
</table>

<table width="600px"><tbody>
</tbody><thead><tr><th colspan="2">Predicted tRNA genes</th></tr></thead>
<tbody><tr><td width="80%">tRNAs decoding Standard 20 AA</td><td>              0</td></tr>
<tr><td width="80%">Selenocysteine tRNAs (TCA)</td><td>                 0</td></tr>
<tr><td width="80%">Possible suppressor tRNAs (CTA,TTA,TCA)</td><td>    0</td></tr>
<tr><td width="80%">tRNAs with undetermined/unknown isotypes</td><td>   0</td></tr>
<tr><td width="80%">Predicted pseudogenes</td><td>                      0</td></tr>
<tr><td width="80%" bgcolor="#e6f2ff">Total tRNAs</td><td bgcolor="#e6f2ff">                                0</td></tr>
</tbody>
</table>
<table>
<thead><tr><th colspan="20">tRNAs with introns:     	0</th></tr></thead>
<tbody>
<tr></tr>
<tr></tr>
<tr></tr>
</tbody>
</table>
<table width="600px">
<thead><tr><th colspan="7">Isotype / Anticodon Counts:</th></tr></thead>
<tbody>
<tr><td bgcolor="#e6f2ff">Ala: 0</td><td>AGC: 0</td><td>GGC: 0</td><td>CGC: 0</td><td>TGC: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Gly: 0</td><td>ACC: 0</td><td>GCC: 0</td><td>CCC: 0</td><td>TCC: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Pro: 0</td><td>AGG: 0</td><td>GGG: 0</td><td>CGG: 0</td><td>TGG: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Thr: 0</td><td>AGT: 0</td><td>GGT: 0</td><td>CGT: 0</td><td>TGT: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Val: 0</td><td>AAC: 0</td><td>GAC: 0</td><td>CAC: 0</td><td>TAC: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Ser: 0</td><td>AGA: 0</td><td>GGA: 0</td><td>CGA: 0</td><td>TGA: 0</td><td>ACT: 0</td><td>GCT: 0</td></tr>
<tr><td bgcolor="#e6f2ff">Arg: 0</td><td>ACG: 0</td><td>GCG: 0</td><td>CCG: 0</td><td>TCG: 0</td><td>CCT: 0</td><td>TCT: 0</td></tr>
<tr><td bgcolor="#e6f2ff">Leu: 0</td><td>AAG: 0</td><td>GAG: 0</td><td>CAG: 0</td><td>TAG: 0</td><td>CAA: 0</td><td>TAA: 0</td></tr>
<tr><td bgcolor="#e6f2ff">Phe: 0</td><td>AAA: 0</td><td>GAA: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Asn: 0</td><td>ATT: 0</td><td>GTT: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Lys: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>CTT: 0</td><td>TTT: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Asp: 0</td><td>ATC: 0</td><td>GTC: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Glu: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>CTC: 0</td><td>TTC: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">His: 0</td><td>ATG: 0</td><td>GTG: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Gln: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>CTG: 0</td><td>TTG: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Ile: 0</td><td>AAT: 0</td><td>GAT: 0</td><td>CAT: 0</td><td>TAT: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Met: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>CAT: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Tyr: 0</td><td>ATA: 0</td><td>GTA: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Supres: 0</td><td>&nbsp;</td><td>CTA: 0</td><td>TTA: 0</td><td>TCA: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Cys: 0</td><td>ACA: 0</td><td>GCA: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">Trp: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>CCA: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td bgcolor="#e6f2ff">SelCys: 0</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>TCA: 0</td><td>&nbsp;</td><td>&nbsp;</td></tr>
</tbody>
</table>


<hr>
<h5><b>tRNAscan-SE command used in this prediction run:</b></h5>
<pre>tRNAscan-SE -qQ --detail -o# -m# -f# -l# -c <a href="http://lowelab.ucsc.edu/download/tRNAscan-SE.conf" target="_blank">tRNAscan-SE.conf</a> --max -O (fasta file)

		</pre></div>
	</div>
</section>
	
<section id="footer">
	<div class="row">
		<div class="small-12 columns">
			<p><a href="http://lowelab.ucsc.edu/" target="_blank">The Lowe Lab</a>, Biomolecular Engineering, University of California Santa Cruz</p>
		</div>
	</div>
</section>

		<a class="exit-off-canvas"></a>
	</div>      
</div>
		
		<script src="MG655623_results_files/backToTop.js"></script>
	  	<script src="MG655623_results_files/foundation.js"></script>
	  	<script type="text/javascript">
	    48 48 1042document).foundation();
	  	</script>
	


</body></html>