
<!-- saved from url=(0060)https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/ -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <title>I-TASSER results</title>
<script type="text/javascript" src="./I-TASSER results_files/JSmol.min.js"></script><script type="text/javascript" src="./I-TASSER results_files/Jmol2.js"></script></head>



<body bgcolor="#FFFFFF">
<font face="Times New Roman">
[<a href="http://zhanglab.ccmb.med.umich.edu/">Home</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">Server</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">Queue</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/about.html">About</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/remove.html">Remove</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/statistics.html">Statistics</a>]
<br>
<table border="0" style="table-layout:fixed;" width="100%" cellpadding="2" cellspaceing="0">
<tbody><tr><td>
  <h1 align="center">I-TASSER results for job id S359535</h1>
<h3 align="center"> [Click on <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/S359535_results.tar.bz2" download="">S359535_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>
<center> (Click on <a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/annotation" target="_blank">Annotation of I-TASSER Output</a> to read the instructions for how to interpret the results on this page)

</center><br>
</td></tr>
<tr><td>
   <div style="background:#6599FF;width:350px;">
   <table><tbody><tr><td style="valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Submitted Sequence in <a href="http://zhanglab.ccmb.med.umich.edu/FASTA/" target="_blank">FASTA format</a></font>
   </td></tr></tbody></table>
   </div>
   <br>
   <div style="position:relative;left:30px;">
   <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
   <tbody><tr><td colspan="2">&gt;seq<br>
MYIFMFFNFYLIISFLLFLIIILFSFLLLFFLYLFNNFYEFSSLSFHSGFYSSIFFLFYI<br>
SLMFFFFSFHNLSWNNLYSEIFFLSLDYFFYILLFYFVFLVTLLFICVFSGLNKKIIRSF<br>
LKSFKNNHGSLYLAYTFNFVFFQISDIYIMYYGCIFYLIAFLVLFILFYLKVPFSPDPKK<br>
VFFFSLLFFVTFTFFTTFHYYVIVYPVIKFPLFCLFFNSILHFVVKIFSEPLEAEVKFEN<br>
FLYFLMGCVFICFIMSFCYIILSKIHIIFMGLINDIFLPGNFLLQAYTEVHSIKALSENW<br>
FWTSWIFYQFEPVYYILINHPEWLGLIFALFYVISHGFLYILLMPFIKKKGKQF<br>
</td></tr>
   </tbody></table>
   </div>
</td></tr>
  <tr><td>
   <br>
   <div style="background:#6599FF;width:300px;">
   <table><tbody><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Secondary Structure</font>
   </td></tr></tbody></table>
   </div>
   <br>
   <div style="position:relative;left:30px;">
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tbody><tr><td valign="bottom"><font="arial"><b>Sequence</b></font="arial"></td><td bgcolor="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;280&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;300&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;320&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;340<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>MYIFMFFNFYLIISFLLFLIIILFSFLLLFFLYLFNNFYEFSSLSFHSGFYSSIFFLFYISLMFFFFSFHNLSWNNLYSEIFFLSLDYFFYILLFYFVFLVTLLFICVFSGLNKKIIRSFLKSFKNNHGSLYLAYTFNFVFFQISDIYIMYYGCIFYLIAFLVLFILFYLKVPFSPDPKKVFFFSLLFFVTFTFFTTFHYYVIVYPVIKFPLFCLFFNSILHFVVKIFSEPLEAEVKFENFLYFLMGCVFICFIMSFCYIILSKIHIIFMGLINDIFLPGNFLLQAYTEVHSIKALSENWFWTSWIFYQFEPVYYILINHPEWLGLIFALFYVISHGFLYILLMPFIKKKGKQF</td></tr>
    <tr><td><font="arial"><span title="Secondary Structure Prediction"><b>Prediction</b></span></font="arial"></td><td bgcolor="#D7D7D7">C<font color="blue">SSSSS</font><font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCC<font color="red">HHHHHHHHHHHH</font>CCCC<font color="blue">SSSSSSSSSSSSSSSSSS</font><font color="red">HHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCCC<font color="blue">SSS</font><font color="red">HHHHHHHHHHHHHHHH</font><font color="blue">SSSSSSSS</font><font color="red">HHHHHHHHHHHHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCC<font color="red">HHHHHHHHHHHHHHHHHH</font>C<font color="red">HHHHHHHHHH</font>CC<font color="blue">SSSSSSS</font>CC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCC</td>
    </tr><tr><td><font="arial"><span title="Confidence Score of Prediction"><b>Conf.Score</b></span></font="arial"></td><td bgcolor="#ECECEC">978742099999999999999999999999999974254311110124368889999999999999975246566788999999899999999999999999999999716558999999998643786299987854899984103135875999999999999999812888979653422577999999986421467898533211369999988999999994753276752997899999699999999999999998999977755401067468999988887298875350578877641240699985787888999999999961089999879888507779</td></tr>
    <tr><td><font="arial"></font="arial"></td><td bgcolor="#ECECEC"><font color="red">H</font>:Helix; <font color="blue">S</font>:Strand; C:Coil</td></tr><tr>
    </tr></tbody></table>
  </div>
  </td></tr>
  <tr><td>
   <br>
   <div style="background:#6599FF;width:300px;">
   <table><tbody><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Solvent Accessibility</font>
   </td></tr></tbody></table>
   </div>
   <br>
   <div style="position:relative;left:30px;">
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tbody><tr><td valign="bottom"><font="arial"><b>Sequence</b></font="arial"></td><td bgcolor="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;280&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;300&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;320&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;340<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>MYIFMFFNFYLIISFLLFLIIILFSFLLLFFLYLFNNFYEFSSLSFHSGFYSSIFFLFYISLMFFFFSFHNLSWNNLYSEIFFLSLDYFFYILLFYFVFLVTLLFICVFSGLNKKIIRSFLKSFKNNHGSLYLAYTFNFVFFQISDIYIMYYGCIFYLIAFLVLFILFYLKVPFSPDPKKVFFFSLLFFVTFTFFTTFHYYVIVYPVIKFPLFCLFFNSILHFVVKIFSEPLEAEVKFENFLYFLMGCVFICFIMSFCYIILSKIHIIFMGLINDIFLPGNFLLQAYTEVHSIKALSENWFWTSWIFYQFEPVYYILINHPEWLGLIFALFYVISHGFLYILLMPFIKKKGKQF</td></tr>
    <tr><td><font="arial"><span title="Solvent Accessibility Prediction"><b>Prediction</b></span></font="arial"></td><td bgcolor="#D7D7D7">310132331211213221211233232211200001201302302121010000001122121110000220314301110000212331102123202212111000011033400330043035430000001000000010000000111011122132212111001011335232000000122210022022000000010131000000011001000200243141413042000000100100201100000023010000000200111020003012303304112731000000011030000000331300000000100222110100000002543675</td></tr><tr>
    </tr><tr><td><font="arial"></font="arial"></td><td bgcolor="#ECECEC">Values range from <b>0</b> (buried residue) to <b>9</b> (highly exposed residue)</td></tr><tr>
    </tr></tbody></table>
  </div>
  </td></tr>
  <tr><td>
   <br>
   <div style="background:#6599FF;width:300px;">
   <table><tbody><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Predicted normalized B-factor</font>
   </td></tr></tbody></table>
   </div>
   <br>

<font size="2">
(B-factor is a value to indicate the extent of the inherent thermal 

mobility of residues/atoms in proteins. In I-TASSER, this value is 

deduced from threading template proteins from the PDB in combination with the 

sequence profiles derived from sequence databases. The reported B-factor 

profile in the figure below corresponds to the normalized B-factor of the 

target protein, defined by B=(B'-u)/s, where B' is the raw B-factor 

value, u and s are respectively the mean and standard deviation of the raw 

B-factors along the sequence. 

<a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/lscore.txt" target="_blank">Click here to read more about predicted 

normalized B-factor</a>) 

</font> 

<br> 

   <br> 

   <div style="position:relative;left:30px;">
    <table border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tbody><tr> <td align="center"> <img src="./I-TASSER results_files/BFP.png">  </td>  </tr>
    </tbody></table>
  </div>
  </td></tr>
  <tr><td>
   <br>
   <div style="background:#6599FF;width:450px;">
   <table><tbody><tr><td style="valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 10 threading templates used by I-TASSER</font>
   </td></tr></tbody></table>
   </div>
   <br>
   
<font size="2"> 

(I-TASSER modeling starts from the structure templates identified by LOMETS from the PDB library. 

LOMETS is a meta-server threading approach containing multiple threading programs, 

where each threading program can generate tens of thousands of 

template alignments. I-TASSER only uses the templates of the highest significance in the threading alignments, 

the significance of which are measured by the Z-score, i.e. the difference between the raw and average scores 

in the unit of standard deviation. The templates in this section are the 10 best templates selected from 

the LOMETS threading programs. Usually, one template of the highest Z-score is selected from each threading program, where the threading programs are sorted by the average performance in the large-scale benchmark test experiments.) 

</font> 

   <br> 

   <br> 

   <div style="position:relative;left:30px;">
   <table style="font-size:13px;background:#F2F2F2;">
      <tbody><tr> <th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br>Hit</font></th><th align="center" valign="top"><font face="Arial">Iden1</font></th><th align="center" valign="top"><font face="Arial">Iden2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br>Z-score</font></th><th valign="top"><font face="Arial">Download<br>Align.</font></th><td>&nbsp;</td><td bgcolor="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;280&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;300&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;320&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;340<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td></tr>
      <tr><td colspan="7"></td><th align="right" valign="bottom"><font face="Arial">Sec.Str<br>Seq</font></th><td bgcolor="#D7D7D7" style="font-family:Monospace;font-size:13px;">C<font color="blue">SSSSS</font><font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCC<font color="red">HHHHHHHHHHHH</font>CCCC<font color="blue">SSSSSSSSSSSSSSSSSS</font><font color="red">HHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCCC<font color="blue">SSS</font><font color="red">HHHHHHHHHHHHHHHH</font><font color="blue">SSSSSSSS</font><font color="red">HHHHHHHHHHHHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCC<font color="red">HHHHHHHHHHHHHHHHHH</font>C<font color="red">HHHHHHHHHH</font>CC<font color="blue">SSSSSSS</font>CC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCC<br>MYIFMFFNFYLIISFLLFLIIILFSFLLLFFLYLFNNFYEFSSLSFHSGFYSSIFFLFYISLMFFFFSFHNLSWNNLYSEIFFLSLDYFFYILLFYFVFLVTLLFICVFSGLNKKIIRSFLKSFKNNHGSLYLAYTFNFVFFQISDIYIMYYGCIFYLIAFLVLFILFYLKVPFSPDPKKVFFFSLLFFVTFTFFTTFHYYVIVYPVIKFPLFCLFFNSILHFVVKIFSEPLEAEVKFENFLYFLMGCVFICFIMSFCYIILSKIHIIFMGLINDIFLPGNFLLQAYTEVHSIKALSENWFWTSWIFYQFEPVYYILINHPEWLGLIFALFYVISHGFLYILLMPFIKKKGKQF</td></tr>       <tr><td align="center"><font face="Arial">1</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5tgz" target="_blank">5tgzA</a></td><td align="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.16</font></td><td><font face="Arial"> 0.70</font></td><td align="center"><font face="Arial"> 1.16</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading1.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">--------AIAVL<font color="#FA9600">S</font>LT<font color="#0F820F">L</font>GTFTV<font color="#0F820F">L</font>ENL<font color="#0F820F">L</font>V<font color="#0F820F">L</font>CVILHSRSLRCRP<font color="#FA9600">S</font>YH<font color="#3232AA">F</font>IGSLVADLLGSVIFVYS<font color="#3232AA">F</font>ID<font color="#3232AA">F</font>HVFHRKDSRNVFLFKLGGVTAS<font color="#3232AA">F</font>TASVGSLFLAAIDRY<font color="#0F820F">I</font>SIHRP<font color="#0F820F">L</font>AY<font color="#145AFF">K</font>R<font color="#0F820F">I</font>VTRP<font color="#145AFF">K</font>AVV--------------------------AF<font color="#FFCC00">C</font>LMWT<font color="#0F820F">I</font><font color="#333333">A</font>IVIAVLPLLGQSVC<font color="#FA9600">S</font>DIFPHIDKTY<font color="#0F820F">L</font>M<font color="#3232AA">F</font>WIGVV-------------SVLLLFIVYAYMY<font color="#0F820F">I</font><font color="#0F820F">L</font>WKAHSHAPDQARMDIELAKT<font color="#0F820F">L</font>VLILVVLI<font color="#0F820F">I</font><font color="#FFCC00">C</font>WGPLLAIMVYDFAFCSMLC<font color="#0F820F">L</font>L<font color="#00DCDC">N</font>STV---<font color="#00DCDC">N</font>PIIY<font color="#333333">A</font>LRSKDLRH<font color="#333333">A</font>FRS--------------------------------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">2</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5aym" target="_blank">5aymA</a></td><td align="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.96</font></td><td align="center"><font face="Arial"> 1.08</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading2.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">------LQVAAFYYLIVKIGTF<font color="#0F820F">L</font>LTPSSGKWIDTHPRIQVVKWGVWLQF<font color="#3232AA">F</font>AILAGMV<font color="#3232AA">F</font>FGM<font color="#0F820F">L</font>DGLVRAGGRE<font color="#FA9600">S</font><font color="#B45AB4">W</font>LLSVLF<font color="#0F820F">I</font>ALALSGVMASLGSQITDIS<font color="#0F820F">V</font>GNDLAPSLVA-PE<font color="#145AFF">K</font>LTHFNSWLRRIDLATEVG<font color="#333333">A</font>PILAGAL<font color="#3232AA">F</font>PLAGLFLIGLWNLVSFVPEYF<font color="#0F820F">L</font>LRNVIQRSGLD<font color="#D69682">P</font>IFWLILSYA<font color="#0F820F">L</font><font color="#0F820F">L</font>WLSVLSPHGVLLAAYLKDEIGL<font color="#3232AA">F</font>RGLGAV<font color="#3232AA">F</font>GL<font color="#0F820F">I</font>STVSFPYLVRR<font color="#0F820F">L</font>GLISSSRWH<font color="#0F820F">L</font>G<font color="#3232AA">F</font>QGVTLG<font color="#0F820F">I</font>AVTAFAMGSTA<font color="#FA9600">S</font>VYVFLGCI<font color="#0F820F">L</font>LSRVG<font color="#0F820F">L</font>GEFE<font color="#0F820F">L</font>R<font color="#00DCDC">Q</font>RLIPEGRRGE<font color="#0F820F">L</font>NSLSSL<font color="#FA9600">T</font>TTSATLILFSAGS<font color="#0F820F">L</font>LPQT<font color="#E60A0A">E</font>----D<font color="#3232AA">F</font>KYLVYV<font color="#FA9600">S</font>LAAVLLANVV<font color="#3232AA">F</font><font color="#0F820F">I</font><font color="#145AFF">K</font>WSSR--</td></tr>
       <tr><td align="center"><font face="Arial">3</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4hea" target="_blank">4heaM</a></td><td align="center"><font face="Arial"> 0.16</font></td><td><font face="Arial"> 0.21</font></td><td><font face="Arial"> 0.82</font></td><td align="center"><font face="Arial"> 2.62</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading3.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">V<font color="#3232AA">Y</font>WAFGLDG---L<font color="#FA9600">S</font>A<font color="#0F820F">L</font>F<font color="#3232AA">F</font><font color="#0F820F">L</font>T<font color="#0F820F">I</font>A<font color="#0F820F">L</font>TV<font color="#3232AA">F</font><font color="#0F820F">L</font>GALV-------------ARVE<font color="#666666">G</font>RFLGLAL<font color="#0F820F">L</font>MEGL<font color="#0F820F">L</font>LGL<font color="#3232AA">F</font>AA<font color="#3232AA">F</font>FEAALIPALLMLYLALYTFVLFT<font color="#0F820F">L</font>VGSLPM<font color="#0F820F">L</font>AAV<font color="#0F820F">L</font>GARLL<font color="#FA9600">S</font><font color="#666666">G</font>SPFWVFLG<font color="#3232AA">F</font>ALA<font color="#3232AA">F</font>AIKTP-----------------LFADAL<font color="#666666">G</font>TLYKVGV<font color="#3232AA">F</font>AFFRFAIP<font color="#0F820F">L</font>APEGFAQAQGLLL<font color="#3232AA">F</font>LAALSALYGAWVA<font color="#3232AA">F</font>AA-----KDF<font color="#145AFF">K</font>TL<font color="#0F820F">L</font>AYAGLSHMGVAALGV<font color="#3232AA">F</font><font color="#FA9600">S</font>GT--P<font color="#E60A0A">E</font>GAMGGLYLLAASG<font color="#0F820F">V</font>YTGGLFLLAGRLYERTG<font color="#0F820F">I</font>L<font color="#3232AA">F</font>LAMV---G<font color="#0F820F">L</font><font color="#D69682">P</font><font color="#666666">G</font>--<font color="#0F820F">L</font>SGFPG<font color="#E60A0A">E</font>FLTLLGAYKASP<font color="#B45AB4">W</font>LAALAF---------------<font color="#0F820F">L</font>SV<font color="#0F820F">I</font>ASAA<font color="#3232AA">Y</font>-----A<font color="#0F820F">L</font>TAFQKT<font color="#3232AA">F</font>WEEG<font color="#666666">G</font>SGV</td></tr>
       <tr><td align="center"><font face="Arial">4</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2a65" target="_blank">2a65A</a></td><td align="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.21</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 1.49</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading4.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">RNAVGLG<font color="#00DCDC">N</font><font color="#3232AA">F</font>VQAAA<font color="#3232AA">F</font>MIYI<font color="#0F820F">I</font>AF<font color="#0F820F">L</font>LVGIP<font color="#0F820F">L</font>MWI<font color="#3232AA">Y</font>GGAQTPAIFY<font color="#0F820F">L</font>LWRNR<font color="#3232AA">F</font>APLVVAIY<font color="#3232AA">Y</font>VYIE<font color="#3232AA">F</font>AIKFLGLVPEDPDSILRP<font color="#3232AA">F</font>KEFL<font color="#3232AA">Y</font>SYIFAYIV<font color="#3232AA">F</font>LITMFINVSILIR<font color="#666666">G</font>IS<font color="#145AFF">K</font>RFAKIAMPTLF----I<font color="#0F820F">L</font>AVFE<font color="#FA9600">T</font>P<font color="#00DCDC">N</font>GTAADG----LNFLLKDPGVWIAA<font color="#0F820F">V</font>GQ<font color="#0F820F">I</font>F<font color="#3232AA">F</font>T<font color="#0F820F">L</font>SLG<font color="#3232AA">F</font><font color="#FA9600">S</font>YVR<font color="#145AFF">K</font>DQDIVLSGLTAATLEKAEV-----------ILGGSISIPAAVA<font color="#3232AA">F</font>A<font color="#0F820F">V</font>A<font color="#0F820F">I</font>AKAG<font color="#0F820F">L</font>P<font color="#333333">A</font>IFT<font color="#3232AA">F</font>LG<font color="#3232AA">F</font><font color="#0F820F">L</font>W<font color="#3232AA">F</font>FLTSSIAIMQPMIAFLEVLWTAA<font color="#0F820F">I</font>VFFSAHLVM<font color="#0F820F">L</font>NKSLEMDFWA<font color="#E60A0A">E</font>LIIFEEINRGGVPRIYYYVEYI<font color="#D69682">P</font>KIM---EETV<font color="#B45AB4">W</font>ITRFYIILFLFLTFLVFLAE-RRRNHESAGT</td></tr>
       <tr><td align="center"><font face="Arial">5</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5tjv" target="_blank">5tjvA</a></td><td align="center"><font face="Arial"> 0.13</font></td><td><font face="Arial"> 0.17</font></td><td><font face="Arial"> 0.68</font></td><td align="center"><font face="Arial"> 1.15</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading5.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">--------AIAVL<font color="#FA9600">S</font>LT<font color="#0F820F">L</font>GTFTV<font color="#0F820F">L</font>ENL<font color="#0F820F">L</font>V<font color="#0F820F">L</font>CVILHSRSLRCRP<font color="#FA9600">S</font>YH<font color="#3232AA">F</font>IGSLVADLLGSVIFVYS<font color="#3232AA">F</font>ID<font color="#3232AA">F</font>HVFHRKDSRNVFLFKLGGVTAS<font color="#3232AA">F</font>TASVGSLFLAAIDRY<font color="#0F820F">I</font>SIHRP<font color="#0F820F">L</font>AY<font color="#145AFF">K</font>R<font color="#0F820F">I</font>VTRP<font color="#145AFF">K</font>AVV--------------------------AF<font color="#FFCC00">C</font>LMWT<font color="#0F820F">I</font><font color="#333333">A</font>IVIAVLPLLGQSVC<font color="#FA9600">S</font>DIFPHIDETY<font color="#0F820F">L</font>M<font color="#3232AA">F</font>WIG--------------------VTSVLLLF<font color="#0F820F">I</font>VYAYMY<font color="#0F820F">I</font>L<font color="#FA9600">S</font>DQARMDIRLAKT<font color="#0F820F">L</font>VLILVVLI<font color="#0F820F">I</font><font color="#FFCC00">C</font>WGPLLAIMVYDVAFCSMLC<font color="#0F820F">L</font>L<font color="#00DCDC">N</font>STV---<font color="#00DCDC">N</font>PIIY<font color="#333333">A</font>LRSKDLRH<font color="#333333">A</font>FRS--------------------------------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">6</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4k1c" target="_blank">4k1cA</a></td><td align="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.21</font></td><td><font face="Arial"> 0.98</font></td><td align="center"><font face="Arial"> 1.03</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading6.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">LLV<font color="#3232AA">F</font>VPLGLIWGHFQ<font color="#0F820F">L</font>SHTLTF<font color="#0F820F">L</font><font color="#3232AA">F</font>N<font color="#3232AA">F</font><font color="#0F820F">L</font>AIIP<font color="#0F820F">L</font>AAILANATEELADKAGNTIGGLLNATFGNAVELIV<font color="#FA9600">S</font>IIA<font color="#0F820F">L</font>KKGQVRIVQASMLGSLLSNL<font color="#0F820F">L</font><font color="#0F820F">L</font>VLGLCFIFGGYNR<font color="#0F820F">V</font>QQTF<font color="#00DCDC">N</font>QTAAQTMSSLLAIACA<font color="#FA9600">S</font><font color="#0F820F">L</font>LIPAA<font color="#3232AA">F</font>RATLPF<font color="#0F820F">I</font>DGKILELSRGTSIV<font color="#0F820F">I</font>LIVYVLF<font color="#0F820F">L</font>YFQLGSHHALFEQQEEETDEVMS<font color="#FA9600">T</font>ISRNPHHS-----LS<font color="#0F820F">V</font>KSSLVIL<font color="#0F820F">L</font>GTTV<font color="#0F820F">I</font>IS<font color="#3232AA">F</font>CADFLVGTIDNV<font color="#0F820F">V</font>ESTGLSKTFI<font color="#666666">G</font>LIV<font color="#0F820F">I</font>PIVGNAAEHVT<font color="#FA9600">S</font>VLVAMKDKMDLALGVAIGSS<font color="#0F820F">L</font><font color="#00DCDC">Q</font>VALF<font color="#0F820F">V</font>TPFMV<font color="#0F820F">L</font>VG-<font color="#B45AB4">W</font>MIDVPMTLN<font color="#3232AA">F</font>STFETATLFIAVF<font color="#0F820F">L</font>SNYLI<font color="#0F820F">L</font>DGESNWLEGVMS<font color="#0F820F">L</font>AMY<font color="#0F820F">I</font>LIAMAF<font color="#3232AA">F</font></td></tr>
       <tr><td align="center"><font face="Arial">7</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mk7" target="_blank">3mk7A</a></td><td align="center"><font face="Arial"> 0.13</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 0.83</font></td><td align="center"><font face="Arial"> 2.58</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading7.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">S-------<font color="#3232AA">F</font>GRLRPLHTNAV<font color="#0F820F">I</font>FA<font color="#3232AA">F</font>GGCA<font color="#0F820F">L</font><font color="#3232AA">F</font>ATSYYSVQRTCQTTL<font color="#3232AA">F</font>APKLAAFT<font color="#3232AA">F</font>WGWQLVILLAAI<font color="#FA9600">S</font>LPLGFVWVA<font color="#3232AA">Y</font>AVV<font color="#3232AA">F</font><font color="#3232AA">F</font>GT<font color="#0F820F">L</font>AKIYVGNW<font color="#3232AA">F</font>FGA<font color="#3232AA">F</font>IL<font color="#FA9600">T</font>VAILH<font color="#0F820F">V</font>VNN<font color="#0F820F">L</font>-------------------EIPV<font color="#FA9600">T</font>AMKSYSL-------<font color="#3232AA">Y</font>A<font color="#666666">G</font>ATDAMVQWWYGHNAVGFFLT-------AG<font color="#3232AA">F</font>LGIMYYFVPKQAERP---<font color="#0F820F">V</font>YS<font color="#3232AA">Y</font>RLSIVHFWA<font color="#0F820F">L</font>ITVY<font color="#0F820F">I</font>WAGPHHLHYTA<font color="#0F820F">L</font>PDWAQSLGMVMS<font color="#0F820F">L</font>ILLAPSWGG<font color="#FFCC00">M</font>INGMMT<font color="#0F820F">L</font><font color="#FA9600">S</font>G---AWHK<font color="#0F820F">L</font>RS<font color="#E60A0A">D</font>PI<font color="#0F820F">L</font>RFLVVSL<font color="#333333">A</font>FYGMSTFEGPMMAIKTVNALSH----------YTHVHA<font color="#666666">G</font>ALGWVAM<font color="#0F820F">V</font>SIGALYHLVPKV<font color="#3232AA">F</font>----<font color="#666666">G</font>REQ</td></tr>
       <tr><td align="center"><font face="Arial">8</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4ikv" target="_blank">4ikvA</a></td><td align="center"><font face="Arial"> 0.19</font></td><td><font face="Arial"> 0.18</font></td><td><font face="Arial"> 0.95</font></td><td align="center"><font face="Arial"> 1.47</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading8.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">VPQRG<font color="#3232AA">F</font><font color="#3232AA">F</font>GHPKGL-<font color="#3232AA">F</font>T<font color="#0F820F">L</font><font color="#3232AA">F</font>FTEFW<font color="#3232AA">F</font><font color="#FA9600">S</font>YRAILVY<font color="#3232AA">Y</font>MYYEVSK-GG<font color="#0F820F">L</font>GL-DEHLALAIMSI<font color="#3232AA">Y</font>GA<font color="#0F820F">L</font>VYMSLADRVFGTSRA---V<font color="#3232AA">F</font>YGG<font color="#0F820F">L</font>LIMAH<font color="#0F820F">I</font>A<font color="#0F820F">L</font>AVAALF<font color="#0F820F">V</font>SMAL<font color="#0F820F">I</font>VLGT<font color="#666666">G</font><font color="#0F820F">L</font>L<font color="#145AFF">K</font>PNVS<font color="#FA9600">S</font>IVDMY<font color="#145AFF">K</font>PGD<font color="#666666">G</font>AFTAGMKY<font color="#00DCDC">N</font><font color="#3232AA">F</font>H-------LGFGLAAVGMFLG-<font color="#0F820F">L</font><font color="#0F820F">V</font>V<font color="#3232AA">F</font>VATN<font color="#0F820F">L</font>GLAGTYV<font color="#D69682">P</font>NPL<font color="#3232AA">F</font>VA<font color="#FA9600">S</font>AM<font color="#3232AA">F</font>WAIQ<font color="#FA9600">T</font>ILANYDKRT<font color="#0F820F">I</font>HLSPAW<font color="#3232AA">F</font>QSLNPL<font color="#3232AA">F</font>II<font color="#0F820F">I</font><font color="#0F820F">L</font>AP<font color="#0F820F">V</font>FAWMWVK<font color="#0F820F">L</font>GKR-QPTIPQK<font color="#3232AA">F</font>AL<font color="#666666">G</font>LL<font color="#3232AA">F</font>AS<font color="#3232AA">F</font><font color="#0F820F">I</font>VILVPGH<font color="#0F820F">L</font><font color="#FA9600">S</font>GVLSY<font color="#3232AA">F</font>IVVLGELC<font color="#0F820F">L</font>-SPVG<font color="#0F820F">L</font>S<font color="#333333">A</font>T<font color="#FA9600">T</font>KLAAFS<font color="#333333">A</font>QTMSLWFL<font color="#FA9600">S</font>NAAA<font color="#00DCDC">Q</font>AINAQLVRFYT<font color="#D69682">P</font><font color="#E60A0A">E</font>NETAY<font color="#3232AA">F</font>GTIGGAAL<font color="#666666">G</font>LILLAIA<font color="#D69682">P</font>R<font color="#0F820F">I</font>GRLM<font color="#145AFF">K</font>--</td></tr>
       <tr><td align="center"><font face="Arial">9</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ayf" target="_blank">3ayfA</a></td><td align="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 0.92</font></td><td align="center"><font face="Arial"> 1.09</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading9.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">--------K<font color="#3232AA">Y</font>FVVVSA<font color="#0F820F">L</font><font color="#3232AA">F</font>FVQTM<font color="#3232AA">F</font>GA<font color="#0F820F">L</font><font color="#0F820F">L</font>AHYYTWIYDILP<font color="#3232AA">F</font>NIAGY<font color="#8282D2">H</font>LQLAIFWIATAWLGMGI<font color="#3232AA">F</font>IAPLQEPKKQG<font color="#0F820F">L</font>LVDLL<font color="#3232AA">F</font>WA<font color="#0F820F">L</font>VVLVGGSMIGQWLIWQIILVVGMLLWLFIVF<font color="#145AFF">R</font>GVKRGL<font color="#145AFF">K</font>RESDKGG<font color="#0F820F">L</font>IHLLFYSAIAVPFF<font color="#3232AA">Y</font><font color="#0F820F">I</font>FWIIHVEGIFEVFAVVVIGF<font color="#0F820F">L</font>L<font color="#0F820F">V</font>QLRLVT<font color="#145AFF">K</font><font color="#145AFF">K</font>STVRA<font color="#0F820F">L</font>Y<font color="#3232AA">F</font>QF<font color="#FA9600">T</font>ILLGSGVIGIGHH<font color="#3232AA">Y</font>YYNGS<font color="#D69682">P</font>EVWIALGAVFSALEV<font color="#0F820F">I</font>---<font color="#D69682">P</font><font color="#0F820F">L</font>TL-LIL<font color="#E60A0A">E</font>AYEQYK<font color="#FFCC00">M</font>MRDGGAKATFWFL<font color="#0F820F">I</font>STAIWNLVGA<font color="#666666">G</font>VFGFLINLPAVSYFEL<font color="#FA9600">T</font>PA<font color="#8282D2">H</font>GHA<font color="#333333">A</font>M-----MGVYGMFAIAVLL<font color="#3232AA">Y</font>S<font color="#0F820F">L</font>---RNIVKWLKFSCWMLNI<font color="#666666">G</font>LAGMVVITLL-------</td></tr>
       <tr><td align="center"><font face="Arial">10</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3rko" target="_blank">3rkoB</a></td><td align="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.88</font></td><td align="center"><font face="Arial"> 2.56</font></td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/threading10.pdb" download=""><font face="Arial">Download</font></a></td><td>&nbsp;</td><td bgcolor="#ECECEG" style="font-family:Monospace;font-size:13px;">IGFNLVLDMLSVVTGVG<font color="#3232AA">F</font><font color="#0F820F">L</font><font color="#0F820F">I</font>HMYA<font color="#FA9600">S</font>WYMSR<font color="#3232AA">F</font>FAYT<font color="#00DCDC">N</font>L<font color="#3232AA">F</font>IASMVVLVLADNL-LLMY<font color="#0F820F">L</font>GWEGVGLCSYLLIGFYYTD--PKNGAAAMKA<font color="#3232AA">F</font>VVTRVGDVFLAFA<font color="#0F820F">L</font>FILYNEL<font color="#666666">G</font>TL--NF<font color="#145AFF">R</font>EMVELAPAHFAD-------------GNNMLMWATLMLLGGAVGKSAYLIARTHGL<font color="#3232AA">F</font>LMT<font color="#D69682">P</font>EVLHLVGIVGA<font color="#0F820F">V</font><font color="#FA9600">T</font>LLLAGFAALV---QTD<font color="#0F820F">I</font><font color="#145AFF">K</font>RV<font color="#0F820F">L</font>AYSTMSQ<font color="#0F820F">I</font>GYMFLALGVQAWD<font color="#333333">A</font>AIFHLMTHA<font color="#3232AA">F</font>FKALL<font color="#3232AA">F</font>LASGSVILACHIPLVYLC<font color="#3232AA">F</font>LVGGAALSALPLVTAGFFSKDEILAGAMA<font color="#00DCDC">N</font>-------------------G<font color="#8282D2">H</font>---IN<font color="#0F820F">L</font>MV<font color="#333333">A</font>GLVGAFMTS<font color="#0F820F">L</font><font color="#3232AA">Y</font>TFR<font color="#FFCC00">M</font>I<font color="#3232AA">F</font><font color="#0F820F">I</font>VFH<font color="#666666">G</font><font color="#145AFF">K</font>EQ</td></tr>
      <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tbody><tr><td valign="top">(a)</td><td align="justify">All the residues are colored in black; however, those residues in template which are identical to the residue in the query sequence are highlighted in color. Coloring scheme is based on the property of amino acids, where polar are brightly coloured while non-polar residues are colored in dark shade. (<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/color.html" target="_blank">more about the colors used</a>)</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">Rank of templates represents the top ten threading templates used by I-TASSER.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify">Ident1 is the percentage sequence identity of the templates in the threading aligned region with the query sequence.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify">Ident2 is the percentage sequence identity of the whole template chains with  query sequence.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the threading alignment and is equal to the number of aligned residues divided by the length of query protein.</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Norm. Z-score is the normalized Z-score of the threading alignments. Alignment with a Normalized Z-score &gt;1 mean a good alignment and vice versa.</td></tr>
        <tr><td valign="top">(g)</td><td align="justify">Download Align. provides the 3D structure of the aligned regions of the threading templates.</td></tr>
        <tr><td valign="top">(h)</td><td align="justify">The top 10 alignments reported above (in order of their ranking) are from the following threading programs:</td></tr>
        <tr><td>&nbsp;</td><td align="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
1: FFAS-3D&nbsp;&nbsp;&nbsp;2: Neff-PPAS&nbsp;&nbsp;&nbsp;3: pGenTHREADER&nbsp;&nbsp;&nbsp;4: PROSPECT2&nbsp;&nbsp;&nbsp;5: FFAS-3D&nbsp;&nbsp;&nbsp;6: Neff-PPAS&nbsp;&nbsp;&nbsp;7: pGenTHREADER&nbsp;&nbsp;&nbsp;8: PROSPECT2&nbsp;&nbsp;&nbsp;9: FFAS-3D&nbsp;&nbsp;&nbsp;10: pGenTHREADER&nbsp;&nbsp;&nbsp;        </td></tr>
      </tbody></table>
      </td></tr></tbody></table>
   </div>
</td></tr>
  <tr><td>
   <br>
   <div style="background:#6599FF;width:380px;">
   <table><tbody><tr><td style="valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Top 5 final models predicted by I-TASSER </font>
   </td></tr></tbody></table>
   </div>
   <br>
  
<font size="2">

(For each target, I-TASSER simulations generate a large ensemble of
structural conformations, called decoys. To select the final models, I-TASSER uses the SPICKER program to cluster
all the decoys based on the pair-wise structure similarity, and reports up to five models which
corresponds to the five largest structure clusters. The confidence of each model is quantitatively
measured by C-score that is calculated based on the significance of threading template alignments
and the convergence parameters of the structure assembly simulations.
C-score is typically in the range of [-5, 2], where a C-score of a higher value signifies a model with a higher 
confidence and vice-versa. TM-score and RMSD are estimated based on C-score and protein length following 
the correlation observed between these qualities. Since the top 5 models are ranked by the cluster size,
it is possible that the lower-rank models have a higher C-score in 
rare cases. Although the first model has a better
quality in most cases, it is also possible that the lower-rank models have a better quality than the
higher-rank models as seen in our benchmark tests. If the I-TASSER simulations converge, it
is possible to have less than 5 clusters generated; this is usually an indication that the
models have a good quality because of the converged simulations.)
<ul>
<li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/cscore.txt" target="_blank">More about C-score</a>
</li><li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/lscore.html">Local structure accuracy profile of the top five models</a>
</li></ul>
</font>
<ul>   <div style="position:relative;left:1px;">
       <table style="table-layout:fixed;width:1780px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
  <tbody><tr> <td colspan="5"> <font size="2">(By right-click on the images, you can export image file or change the configurations, e.g. modifying the background color or stopping the spin of
your models)</font></td></tr>
       <tr>
       <td align="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model1.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><div id="jmolApplet0_appletinfotablediv" style="width:350px;height:350px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet0_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="350" height="350" id="jmolApplet0_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet0_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet0_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet0_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet0_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet0"</span><span id="jmolApplet0_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet0,false)">[x]</a></span></div><div id="jmolApplet0_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model1.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/core/corebio.z.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/J/render/MeshRenderer.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by J.renderbio.BioShapeRenderer  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/J/shape/Mesh.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by J.renderbio.BioShapeRenderer  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model1.pdb): 7787 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6147 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet1 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet1__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet1__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet1","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet1__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet1_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet1 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet1__592082709420247__ ready</span></div></div></div></div><span id="span_jmolButton0"><input type="button" name="jmolButton0" id="jmolButton0" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox0"><input type="checkbox" name="jmolCheckbox0" id="jmolCheckbox0" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox0">Spin On/Off</label></span></td>
       <td align="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model2.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><div id="jmolApplet1_appletinfotablediv" style="width:350px;height:350px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet1_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="350" height="350" id="jmolApplet1_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet1_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet1_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet1_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet1_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet1"</span><span id="jmolApplet1_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet1,false)">[x]</a></span></div><div id="jmolApplet1_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model2.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model2.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model2.pdb): 2488 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6147 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet2 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet2__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet2__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet2","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet2__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet2_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet2 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet2__592082709420247__ ready</span></div></div></div></div><span id="span_jmolButton1"><input type="button" name="jmolButton1" id="jmolButton1" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox1"><input type="checkbox" name="jmolCheckbox1" id="jmolCheckbox1" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox1">Spin On/Off</label></span></td>
       <td align="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model3.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><div id="jmolApplet2_appletinfotablediv" style="width:350px;height:350px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet2_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="350" height="350" id="jmolApplet2_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet2_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet2_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet2_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet2_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet2"</span><span id="jmolApplet2_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet2,false)">[x]</a></span></div><div id="jmolApplet2_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model3.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model3.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model3.pdb): 2679 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6154 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6154 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/core/corescriptcmd.z.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet3 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet3__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet3__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet3","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet3__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet3_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet3 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet3__592082709420247__ ready</span></div></div></div></div><span id="span_jmolButton2"><input type="button" name="jmolButton2" id="jmolButton2" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox2"><input type="checkbox" name="jmolCheckbox2" id="jmolCheckbox2" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox2">Spin On/Off</label></span></td>
       <td align="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model4.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><div id="jmolApplet3_appletinfotablediv" style="width:350px;height:350px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet3_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="350" height="350" id="jmolApplet3_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet3_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet3_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet3_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet3_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet3"</span><span id="jmolApplet3_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet3,false)">[x]</a></span></div><div id="jmolApplet3_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model4.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model4.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model4.pdb): 2497 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6154 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6154 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/J/g3d/HermiteRenderer.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet4 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet4__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet4__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet4","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet4__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet4_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet4 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet4__592082709420247__ ready</span></div></div></div></div><span id="span_jmolButton3"><input type="button" name="jmolButton3" id="jmolButton3" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox3"><input type="checkbox" name="jmolCheckbox3" id="jmolCheckbox3" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox3">Spin On/Off</label></span></td>
       <td align="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model5.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><div id="jmolApplet4_appletinfotablediv" style="width:350px;height:350px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet4_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="350" height="350" id="jmolApplet4_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet4_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet4_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet4_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet4_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet4"</span><span id="jmolApplet4_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet4,false)">[x]</a></span></div><div id="jmolApplet4_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model5.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model5.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model5.pdb): 2278 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6154 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6154 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">protein</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6154 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet5 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet5__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet5__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet5","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet5__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet5_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet5 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet5__592082709420247__ ready</span></div></div></div></div><span id="span_jmolButton4"><input type="button" name="jmolButton4" id="jmolButton4" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox4"><input type="checkbox" name="jmolCheckbox4" id="jmolCheckbox4" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox4">Spin On/Off</label></span></td>
       </tr> <tr> <td align="left">
<li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1.pdb" download="">Download Model 1</a>

</li><li>C-score=-2.05 (<a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/cscore.txt" target="_blank">Read more about C-score</a>) 

</li><li>Estimated TM-score = 0.470.15

</li><li>Estimated RMSD = 11.44.5

<br><br>

</li></td>
 <td align="left">
<li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model2.pdb" download="">Download Model 2</a>

</li><li>C-score = -3.05 <br><br>

</li></td>
 <td align="left">
<li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model3.pdb" download="">Download Model 3</a>

</li><li>C-score = -3.52 <br><br>

</li></td>
 <td align="left">
<li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model4.pdb" download="">Download Model 4</a>

</li><li>C-score = -3.46 <br><br>

</li></td>
 <td align="left">
<li><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model5.pdb" download="">Download Model 5</a>

</li><li>C-score = -3.89 <br><br>

</li></td>
       </tr>
       </tbody></table>
    </div>
    
  </ul></td></tr>
  
  <tr><td>
   <br>
   <div style="background:#6599FF;width:670px;">
      <table>
         <tbody><tr><td style="valign:middle;"><font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Proteins structurally close to the target in the PDB (as identified by <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">TM-align</a>)</font></td></tr>
      </tbody></table>
   </div>
   <br>
   
<font size="2">

(After the structure assembly simulation, I-TASSER uses the TM-align structural alignment program to match the first I-TASSER model to all structures in the PDB library. This section reports the top 10 proteins from the PDB that have the closest structural similarity, 

i.e. the highest <a href="http://zhanglab.ccmb.med.umich.edu/TM-score">TM-score</a>, to the predicted I-TASSER model. Due to the structural similarity, these proteins often have similar function to the target. However, users are encouraged to use the data in the next section 'Predicted function using COACH' to infer the function of the target protein, since COACH has been extensively trained to derive biological functions from multi-source of sequence and structure features which has on average a higher accuracy than the function annotations derived only from the global structure comparison.)</font> 

   <br>

   <br>

   <div>
   <div style="position:relative;left:30px;background:#F2F2F2;">
       <table style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
           <tbody><tr>
           <td align="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1_5aymA.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;");</script><div id="jmolApplet5_appletinfotablediv" style="width:400px;height:400px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet5_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="400" height="400" id="jmolApplet5_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet5_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet5_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet5_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet5_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet5"</span><span id="jmolApplet5_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet5,false)">[x]</a></span></div><div id="jmolApplet5_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model1_5aymA.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_5aymA.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model1_5aymA.pdb): 3957 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 9161 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">9161 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">NOTE: Backbone amide hydrogen positions are present and will be ignored. Their positions will be approximated, as in standard DSSP analysis.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Use SET dsspCalculateHydrogenAlways FALSE to not use this approximation.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet6 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet6__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet6__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet6","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet6__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet6_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet6 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet6__592082709420247__ ready</span></div></div></div></div><br><script>jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><span id="span_jmolButton5"><input type="button" name="jmolButton5" id="jmolButton5" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox5"><input type="checkbox" name="jmolCheckbox5" id="jmolCheckbox5" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox5">Spin On/Off</label></span></td>
           <td valign="top"><b>Top 10 Identified stuctural analogs in PDB</b><br><br>
                 <font face="Monospace">
                       </font><font face="Monospace"><font face="Monospace">
                       </font></font><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace">
                       </font></font></font></font><table border="0" style="font-family:Arial;font-size:13px;">
                    <tbody><tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th>Rank</th><th>PDB Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov</th><th>Alignment</th></tr>
               
                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5aymA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", true);</script><span id="span_jmolRadioGroup0_0"><input name="jmolRadioGroup0" id="jmolRadioGroup0_0" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()" checked="true"><label for="jmolRadioGroup0_0"></label></span></td><td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5aym" target="_blank">5aymA</a></td><td align="center">0.941</td><td valign="middle" align="center"> 1.21</td><td valign="middle" align="center">0.095</td><td valign="middle">0.958</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_5aymA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4gbyA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_1"><input name="jmolRadioGroup0" id="jmolRadioGroup0_1" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_1"></label></span></td><td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4gby" target="_blank">4gbyA</a></td><td align="center">0.783</td><td valign="middle" align="center"> 3.85</td><td valign="middle" align="center">0.081</td><td valign="middle">0.963</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_4gbyA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4ybqA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_2"><input name="jmolRadioGroup0" id="jmolRadioGroup0_2" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_2"></label></span></td><td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4ybq" target="_blank">4ybqA</a></td><td align="center">0.765</td><td valign="middle" align="center"> 3.39</td><td valign="middle" align="center">0.076</td><td valign="middle">0.915</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_4ybqA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5c65A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_3"><input name="jmolRadioGroup0" id="jmolRadioGroup0_3" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_3"></label></span></td><td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5c65" target="_blank">5c65A</a></td><td align="center">0.758</td><td valign="middle" align="center"> 3.38</td><td valign="middle" align="center">0.101</td><td valign="middle">0.910</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_5c65A.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3o7qA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_4"><input name="jmolRadioGroup0" id="jmolRadioGroup0_4" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_4"></label></span></td><td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3o7q" target="_blank">3o7qA</a></td><td align="center">0.758</td><td valign="middle" align="center"> 3.02</td><td valign="middle" align="center">0.078</td><td valign="middle">0.878</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_3o7qA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3wdoA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_5"><input name="jmolRadioGroup0" id="jmolRadioGroup0_5" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_5"></label></span></td><td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3wdo" target="_blank">3wdoA</a></td><td align="center">0.745</td><td valign="middle" align="center"> 3.19</td><td valign="middle" align="center">0.059</td><td valign="middle">0.881</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_3wdoA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4m64A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_6"><input name="jmolRadioGroup0" id="jmolRadioGroup0_6" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_6"></label></span></td><td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4m64" target="_blank">4m64A</a></td><td align="center">0.720</td><td valign="middle" align="center"> 4.16</td><td valign="middle" align="center">0.068</td><td valign="middle">0.915</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_4m64A.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4pypA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_7"><input name="jmolRadioGroup0" id="jmolRadioGroup0_7" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_7"></label></span></td><td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4pyp" target="_blank">4pypA</a></td><td align="center">0.698</td><td valign="middle" align="center"> 4.13</td><td valign="middle" align="center">0.083</td><td valign="middle">0.901</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_4pypA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4j05A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_8"><input name="jmolRadioGroup0" id="jmolRadioGroup0_8" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_8"></label></span></td><td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4j05" target="_blank">4j05A</a></td><td align="center">0.689</td><td valign="middle" align="center"> 4.22</td><td valign="middle" align="center">0.051</td><td valign="middle">0.890</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_4j05A.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4zowA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_9"><input name="jmolRadioGroup0" id="jmolRadioGroup0_9" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_9"></label></span></td><td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4zow" target="_blank">4zowA</a></td><td align="center">0.686</td><td valign="middle" align="center"> 4.18</td><td valign="middle" align="center">0.059</td><td valign="middle">0.878</td><td align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1_4zowA.pdb" download=""><font face="Arial">Download</font></a></td></tr>
               </tbody></table><font face="Monospace"><font face="Monospace"><font face="Monospace"><br>
               <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
                <tbody><tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
                <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
                <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
                <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
                <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
               </tbody></table>
          </font></font></font></td></tr>
       </tbody></table>
    </div>
    </div>
  </td></tr>
  <tr><td>
  <br><br>
   <div style="background:#6599FF;width:450px;">
   <table><tbody><tr><td style="valign:middle;">
      <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted function using <a href="http://zhanglab.ccmb.med.umich.edu/COFACTOR" target="_blank">COFACTOR</a> and <a href="http://zhanglab.ccmb.med.umich.edu/COACH" target="_blank">COACH</a></font>
   </td></tr></tbody></table>
   </div>
  <br>
  
<font size="2">

(This section reports biological annotations of the target 
protein by COFACTOR and COACH based on the I-TASSER structure prediction. 
While COFACTOR deduces protein functions (ligand-binding sites, EC and GO)
using structure comparison and protein-protein networks,
COACH is a meta-server approach that combines multiple function
annotation results (on ligand-binding sites) from the COFACTOR, 
TM-SITE and S-SITE programs.)

</font>

<br>

<br>

  </td></tr><tr><td>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tbody><tr><td valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Ligand binding sites</font>
      </td></tr></tbody></table>
   </div>
  <br>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tbody><tr><td valign="top" align="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load CH_CF_4d2cA_BS04_78N.pdb; calculate structure rama; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 16,20,21,24; wireframe 75; spacefill 100; color blue;select 16 and *.ca; label %m%R; color label magenta;select 20 and *.ca; label %m%R; color label magenta;select 21 and *.ca; label %m%R; color label magenta;select 24 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;     set spin x 10; set spin y 10; spin off;");</script><div id="jmolApplet6_appletinfotablediv" style="width:400px;height:400px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet6_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="400" height="400" id="jmolApplet6_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet6_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet6_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet6_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet6_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet6"</span><span id="jmolApplet6_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet6,false)">[x]</a></span></div><div id="jmolApplet6_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(CH_CF_4d2cA_BS04_78N.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_CF_4d2cA_BS04_78N.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(CH_CF_4d2cA_BS04_78N.pdb): 3163 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6169 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6169 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/J/thread/SpinThread.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6154 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet7 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet7__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet7__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet7","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet7__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet7_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet7 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet7__592082709420247__ ready</span></div></div></div></div><br><script> jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</script><span id="span_jmolButton6"><input type="button" name="jmolButton6" id="jmolButton6" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox6"><input type="checkbox" name="jmolCheckbox6" id="jmolCheckbox6" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox6">Spin On/Off</label></span>
</td>
       <td valign="top">       <font face="Monospace"></font><font face="Monospace"><font face="Monospace"></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace"></font></font></font></font><font face="Monospace"><font face="Monospace"><font face="Monospace"><font face="Monospace"></font></font></font></font><table border="0" style="font-family:Arial;font-size:13px;"><tbody><tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th>Rank</th><th>C-score</th><th>Cluster<br>size</th> <th>PDB<br>Hit</th> <th>Lig<br>Name</th> <th>Download<br>Complex</th> <th align="left">Ligand Binding Site Residues</th> </tr>
       <tr>
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_4d2cA_BS04_78N.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 16,20,21,24; color blue; wireframe 75; spacefill 100;select 16 and *.ca; label %m%R; color label magenta;select 20 and *.ca; label %m%R; color label magenta;select 21 and *.ca; label %m%R; color label magenta;select 24 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", true);</script><span id="span_jmolRadioGroup0_10"><input name="jmolRadioGroup0" id="jmolRadioGroup0_10" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()" checked="true"><label for="jmolRadioGroup0_10"></label></span></td>
    <td valign="middle" align="center">1</td><td valign="middle" align="center">0.06</td> <td valign="middle" align="center">3</td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch.cgi?pdbid=4d2c" target="_blank">4d2cA</a></td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=78N" target="_blank">78N</a> </td>  <td valign="middle" align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_CF_4d2cA_BS04_78N.pdb" download="">Rep</a>, <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_complex1.pdb" download="">Mult</a></td>  <td valign="middle" align="left">16,20,21,24</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_5eqiA_BS01_5RH.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 98,124,127,187,279,281,301,304,305,308; color blue; wireframe 75; spacefill 100;select 98 and *.ca; label %m%R; color label magenta;select 124 and *.ca; label %m%R; color label magenta;select 127 and *.ca; label %m%R; color label magenta;select 187 and *.ca; label %m%R; color label magenta;select 279 and *.ca; label %m%R; color label magenta;select 281 and *.ca; label %m%R; color label magenta;select 301 and *.ca; label %m%R; color label magenta;select 304 and *.ca; label %m%R; color label magenta;select 305 and *.ca; label %m%R; color label magenta;select 308 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_11"><input name="jmolRadioGroup0" id="jmolRadioGroup0_11" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_11"></label></span></td>
    <td valign="middle" align="center">2</td><td valign="middle" align="center">0.04</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch.cgi?pdbid=5eqi" target="_blank">5eqiA</a></td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=5RH" target="_blank">5RH</a> </td>  <td valign="middle" align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_TM_5eqiA_BS01_5RH.pdb" download="">Rep</a>, <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_complex2.pdb" download="">Mult</a></td>  <td valign="middle" align="left">98,124,127,187,279,281,301,304,305,308</td>    </tr>
       <tr>
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_5d58A_BS21_78M.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 10,13,17,18; color blue; wireframe 75; spacefill 100;select 10 and *.ca; label %m%R; color label magenta;select 13 and *.ca; label %m%R; color label magenta;select 17 and *.ca; label %m%R; color label magenta;select 18 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_12"><input name="jmolRadioGroup0" id="jmolRadioGroup0_12" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_12"></label></span></td>
    <td valign="middle" align="center">3</td><td valign="middle" align="center">0.04</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch.cgi?pdbid=5d58" target="_blank">5d58A</a></td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=78M" target="_blank">78M</a> </td>  <td valign="middle" align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_CF_5d58A_BS21_78M.pdb" download="">Rep</a>, <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_complex3.pdb" download="">Mult</a></td>  <td valign="middle" align="left">10,13,17,18</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_1y66B_BS01_DIO.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 93,96,100; color blue; wireframe 75; spacefill 100;select 93 and *.ca; label %m%R; color label magenta;select 96 and *.ca; label %m%R; color label magenta;select 100 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_13"><input name="jmolRadioGroup0" id="jmolRadioGroup0_13" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_13"></label></span></td>
    <td valign="middle" align="center">4</td><td valign="middle" align="center">0.04</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch.cgi?pdbid=1y66" target="_blank">1y66B</a></td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=DIO" target="_blank">DIO</a> </td>  <td valign="middle" align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_TM_1y66B_BS01_DIO.pdb" download="">Rep</a>, <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_complex4.pdb" download="">Mult</a></td>  <td valign="middle" align="left">93,96,100</td>    </tr>
       <tr>
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_5c65A_BS07_Y01.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 26,189,303,306,311; color blue; wireframe 75; spacefill 100;select 26 and *.ca; label %m%R; color label magenta;select 189 and *.ca; label %m%R; color label magenta;select 303 and *.ca; label %m%R; color label magenta;select 306 and *.ca; label %m%R; color label magenta;select 311 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_14"><input name="jmolRadioGroup0" id="jmolRadioGroup0_14" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_14"></label></span></td>
    <td valign="middle" align="center">5</td><td valign="middle" align="center">0.04</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch.cgi?pdbid=5c65" target="_blank">5c65A</a></td> <td valign="middle" align="left"><a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=Y01" target="_blank">Y01</a> </td>  <td valign="middle" align="center"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_TM_5c65A_BS07_Y01.pdb" download="">Rep</a>, <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/CH_complex5.pdb" download="">Mult</a></td>  <td valign="middle" align="left">26,189,303,306,311</td>    </tr>
       </tbody></table><font face="Monospace"><font face="Monospace"><font face="Monospace">
       <br>
       <br><table cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
       <tbody><tr><td valign="top"></td><td align="justify"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/prob.txt" target="_blank">Download</a> the residue-specific ligand binding probability, which is estimated by SVM. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/Bsites.inf" target="_blank">Download</a> the all possible binding ligands and detailed prediction summary. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/Bsites.clr" target="_blank">Download</a> the templates clustering results. </td></tr> 
        <tr><td valign="top">(a)</td><td align="justify"><b>C-score</b> is the confidence score of the prediction. C-score ranges [0-1], where a higher score indicates a more reliable prediction. </td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><b>Cluster size</b> is the total number of templates in a cluster. </td></tr>
       <tr><td valign="top">(c)</td><td align="justify"><b>Lig Name</b> is name of possible binding ligand. Click the name to view its information in <a href="http://zhanglab.ccmb.med.umich.edu/BioLiP/">the BioLiP database</a>.</td></tr>
       <tr><td valign="top">(d)</td><td align="justify"><b>Rep</b> is a single complex structure with the most representative ligand in the cluster, i.e., the one listed in the <b>Lig Name</b> column. <br>
 <b>Mult</b> is the complex structures with all potential binding ligands in the cluster.</td></tr>
       </tbody></table></font></font></font></td></tr>
     </tbody></table>
  </div>
</td></tr>
  <tr><td>
  <br>
   <div style="position:relative;left:30px;background:#FF9900; width:430px;">
   <table><tbody><tr><td style="valign:middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Enzyme Commission (EC) numbers and active sites</font>
   </td></tr></tbody></table>
   </div>
   <br>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tbody><tr><td align="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; cartoons;      set spin x 10; set spin y 10; spin off;");</script><div id="jmolApplet7_appletinfotablediv" style="width:400px;height:400px;position:relative;font-size:14px;text-align:left"><div id="jmolApplet7_appletdiv" style="z-index: 9000; width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: block;"><canvas width="400" height="400" id="jmolApplet7_canvas2d" style="width: 100%; height: 100%; z-index: 9002;"></canvas></div><div id="jmolApplet7_2dappletdiv" style="position:absolute;width:100%;height:100%;overflow:hidden;display:none"></div><div id="jmolApplet7_infotablediv" style="width: 100%; height: 100%; position: absolute; top: 0px; left: 0px; display: none;"><div id="jmolApplet7_infoheaderdiv" style="height: 20px; width: 100%; display: none; background: yellow;"><span id="jmolApplet7_infoheaderspan">Jmol._Canvas2D (Jmol) "jmolApplet7"</span><span id="jmolApplet7_infocheckboxspan" style="position:absolute;text-align:right;right:1px;"><a href="javascript:Jmol.showInfo(jmolApplet7,false)">[x]</a></span></div><div id="jmolApplet7_infodiv" style="position:absolute;top:20px;bottom:0px;width:100%;height:100%;overflow:auto"><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/core/package.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=false async=true</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/core/core.z.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=false async=true</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">JSmol exec jmolApplet0 start applet null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol JavaScript applet jmolApplet0__592082709420247__ initializing</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue debug null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue logLevel null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue allowjavascript null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">AppletRegistry.checkIn(jmolApplet0__592082709420247__)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">vwrOptions: </span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">{ "name":"jmolApplet0","applet":true,"documentBase":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/","platform":"J.awtjs2d.Platform","fullName":"jmolApplet0__592082709420247__","codePath":"https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/../../bin/jmol//j2s/","display":"jmolApplet0_canvas2d","signedApplet":"true","appletReadyCallback":"Jmol._readyCallback","statusListener":"[J.appletjs.Jmol object]","syncId":"592082709420247","bgcolor":"#000000" }</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">setting document base to "https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">(C) 2012 Jmol Development</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol Version: 14.2.13_2015.03.07  2015-03-07 10:45</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.vendor: Java2Script (HTML5)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">java.version: 2015-03-01 07:13:29 (JSmol/j2s)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">os.name: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2700.0 Iron Safari/537.36</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Access: ALL</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">memory: 0.0/0.0</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">processors available: 1</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">useCommandThread: false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">appletId:jmolApplet0 (signed)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/core/corescript.z.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue emulate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">defaults = "Jmol"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue boxbgcolor null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue bgcolor #000000</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">backgroundColor = "#000000"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ANIMFRAMECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue APPLETREADYCallback Jmol._readyCallback</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">APPLETREADYCallback = "Jmol._readyCallback"</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">callback set for APPLETREADYCallback Jmol._readyCallback APPLETREADY</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ATOMMOVEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue CLICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue DRAGDROPCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ECHOCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue ERRORCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue EVALCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue HOVERCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue IMAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue LOADSTRUCTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MEASURECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MESSAGECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue MINIMIZATIONCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SERVICECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue PICKCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue RESIZECallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SCRIPTCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue SYNCCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue STRUCTUREMODIFIEDCallback null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue doTranslate null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">language=en_US</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue popupMenu null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue script null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue loadInline null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol getValue load null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol applet jmolApplet0__592082709420247__ ready</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">script 1 started</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager.getAtomSetCollectionFromFile(model1.pdb)</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">FileManager opening 1 https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/model1.pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">The Resolver thinks Pdb</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Time for openFile(model1.pdb): 183 ms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">reading 6147 atoms</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: haveSymmetry:false haveUnitcells:false haveFractionalCoord:false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 model in this collection. Use getProperty "modelInfo" or getProperty "auxiliaryInfo" to inspect them.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Default Van der Waals type for model set to Babel</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms created</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ModelSet: autobonding; use  autobond=false  to not generate bonds automatically</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol 14.2.13_2015.03.07  2015-03-07 10:45 DSSP analysis for model 1.1 - null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">W. Kabsch and C. Sander, Biopolymers, vol 22, 1983, pp 2577-2637</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">We thank Wolfgang Kabsch and Chris Sander for writing the DSSP software,</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">and we thank the CMBI for maintaining it to the extent that it was easy to</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">re-engineer for our purposes. At this point in time, we make no guarantee</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">that this code gives precisely the same analysis as the code available via license</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">from CMBI at http://swift.cmbi.ru.nl/gv/dssp</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">All bioshapes have been deleted and must be regenerated.</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">null</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6154 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">9161 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">3014 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6147 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">6169 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">ambientPercent = 40</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">77 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">&nbsp;&nbsp;&nbsp;&nbsp;../../bin/jmol//j2s/core/coretext.z.js</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;"> -- required by ClazzNode  ajax=true async=false</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinFPS is set too fast (30) -- can't keep up!</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">1 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">22 atoms selected</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">solventProbeRadius = 0.9</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinX = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">spinY = 10</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Script completed</span></div><div style="white-space: nowrap;"><span style="white-space: nowrap;">Jmol script terminated</span></div></div></div></div><br><script>jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("spin on", "spin off", "Spin On/Off", false)</script><span id="span_jmolButton7"><input type="button" name="jmolButton7" id="jmolButton7" value="Reset to initial orientation" onclick="Jmol.controls._click(this)" onmouseover="Jmol.controls._mouseOver(this);return true" onmouseout="Jmol.controls._mouseOut()"></span><span id="span_jmolCheckbox7"><input type="checkbox" name="jmolCheckbox7" id="jmolCheckbox7" onclick="Jmol.controls._cbClick(this)" onmouseover="Jmol.controls._cbOver(this);return true" onmouseout="Jmol.controls._mouseOut()"><label for="jmolCheckbox7">Spin On/Off</label></span></td>
           <td valign="top">
             <table border="0" style="font-family:Arial;font-size:13px;"><tbody><tr bgcolor="#DEDEDE"> <th>Click<br>to view</th><th>Rank</th><th>Cscore<sup>EC</sup></th><th>PDB<br>Hit</th><th>TM-score</th><th>RMSD<sup>a</sup></th><th>IDEN<sup>a</sup></th><th>Cov</th><th>EC Number</th><th align="left">Active Site Residues</th></tr>
         <tr>
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_15"><input name="jmolRadioGroup0" id="jmolRadioGroup0_15" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_15"></label></span></td>
         <td align="left"><font face="Arial">1</font></td><td align="center"><font face="Arial">0.112</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2a06" target="_blank">2a06C</a></td><td align="center"><font face="Arial">0.413</font></td><td align="center"><font face="Arial"> 6.00</font></td><td align="center"><font face="Arial">0.087</font></td><td align="right"><font face="Arial">0.638</font></td>
         <td align="left"><span title="Ubiquinol--cytochrome-c reductase."><a href="http://ca.expasy.org/enzyme/1.10.2.2" target="_blank"><font face="Arial">1.10.2.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_16"><input name="jmolRadioGroup0" id="jmolRadioGroup0_16" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_16"></label></span></td>
         <td align="left"><font face="Arial">2</font></td><td align="center"><font face="Arial">0.109</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3cxh" target="_blank">3cxhN</a></td><td align="center"><font face="Arial">0.409</font></td><td align="center"><font face="Arial"> 6.09</font></td><td align="center"><font face="Arial">0.059</font></td><td align="right"><font face="Arial">0.644</font></td>
         <td align="left"><span title="Ubiquinol--cytochrome-c reductase."><a href="http://ca.expasy.org/enzyme/1.10.2.2" target="_blank"><font face="Arial">1.10.2.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr>
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_17"><input name="jmolRadioGroup0" id="jmolRadioGroup0_17" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_17"></label></span></td>
         <td align="left"><font face="Arial">3</font></td><td align="center"><font face="Arial">0.109</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1xme" target="_blank">1xmeA</a></td><td align="center"><font face="Arial">0.384</font></td><td align="center"><font face="Arial"> 7.20</font></td><td align="center"><font face="Arial">0.040</font></td><td align="right"><font face="Arial">0.681</font></td>
         <td align="left"><span title="Cytochrome-c oxidase."><a href="http://ca.expasy.org/enzyme/1.9.3.1" target="_blank"><font face="Arial">1.9.3.1</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_18"><input name="jmolRadioGroup0" id="jmolRadioGroup0_18" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_18"></label></span></td>
         <td align="left"><font face="Arial">4</font></td><td align="center"><font face="Arial">0.109</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2pfl" target="_blank">2pflA</a></td><td align="center"><font face="Arial">0.398</font></td><td align="center"><font face="Arial"> 6.83</font></td><td align="center"><font face="Arial">0.045</font></td><td align="right"><font face="Arial">0.686</font></td>
         <td align="left"><span title="Formate C-acetyltransferase."><a href="http://ca.expasy.org/enzyme/2.3.1.54" target="_blank"><font face="Arial">2.3.1.54</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr>
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script><span id="span_jmolRadioGroup0_19"><input name="jmolRadioGroup0" id="jmolRadioGroup0_19" type="radio" onclick="Jmol.controls._click(this);return true;" onmouseover="Jmol.controls._mouseOver(this);return true;" onmouseout="Jmol.controls._mouseOut()"><label for="jmolRadioGroup0_19"></label></span></td>
         <td align="left"><font face="Arial">5</font></td><td align="center"><font face="Arial">0.108</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2fkn" target="_blank">2fknB</a></td><td align="center"><font face="Arial">0.394</font></td><td align="center"><font face="Arial"> 6.41</font></td><td align="center"><font face="Arial">0.052</font></td><td align="right"><font face="Arial">0.627</font></td>
         <td align="left"><span title="Urocanate hydratase."><a href="http://ca.expasy.org/enzyme/4.2.1.49" target="_blank"><font face="Arial">4.2.1.49</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         </tbody></table><br>
         <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;">
          <tbody><tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
           <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the EC number prediction. Cscore<sup>EC</sup> values range in between [0-1]; <br>where a higher score indicates a more reliable EC number prediction.</td></tr>
           <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
           <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
           <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
           <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided <br>by length of the query protein.</td></tr>
         </tbody></table>
        </td></tr></tbody></table>
  </div>
</td></tr>
  <tr><td>
  <br>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tbody><tr><td valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Gene Ontology (GO) terms</font>
      </td></tr></tbody></table>
   </div>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tbody><tr background:#ff9900;=""><th colspan="8" align="left"><font face="Arial" size="3">Top 10 homologous GO templates in PDB</font></th><th></th><td>&nbsp;</td></tr>
  <tr bgcolor="#DEDEDE"><th><font face="Arial">Rank</font></th><th><font face="Arial">Cscore<sup>GO</sup></font></th><th><font face="Arial">TM-score</font></th><th><font face="Arial">RMSD<sup>a</sup></font></th><th><font face="Arial">IDEN<sup>a</sup></font></th><th><font face="Arial">Cov</font></th><th align="left"><font face="Arial">PDB Hit</font></th><th align="left"><font face="Arial">Associated GO Terms</font></th></tr>
			<tr><td align="center" valign="middle"><font face="Arial">1</font></td><td align="center" valign="middle"><font face="Arial"> 0.17</font></td><td valign="middle" align="center"><font face="Arial">0.7578</font></td><td align="center" valign="middle"><font face="Arial"> 3.02</font></td><td align="center" valign="middle"><font face="Arial"> 0.08</font></td><td valign="middle" align="center"><font face="Arial"> 0.88</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3o7q" target="_blank">3o7qA</a></td><td valign="middle"><font face="Arial">
<span title="fucose metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006004" target="_blank">GO:0006004</a></span>
<span title="fucose transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015756" target="_blank">GO:0015756</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015293" target="_blank">GO:0015293</a></span>
<span title="fructose transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015755" target="_blank">GO:0015755</a></span>
<span title="carbohydrate transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008643" target="_blank">GO:0008643</a></span>
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="hexose:hydrogen symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009679" target="_blank">GO:0009679</a></span>
<span title="galactose transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015757" target="_blank">GO:0015757</a></span>
<span title="galactose:hydrogen symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015517" target="_blank">GO:0015517</a></span>
<span title="arabinose:hydrogen symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015518" target="_blank">GO:0015518</a></span>
<span title="transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055085" target="_blank">GO:0055085</a></span>
<span title="fucose:hydrogen symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015535" target="_blank">GO:0015535</a></span>
<span title="plasma membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" target="_blank">GO:0005886</a></span>
<span title="carbohydrate metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005975" target="_blank">GO:0005975</a></span>
<span title="arabinose transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015751" target="_blank">GO:0015751</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">2</font></td><td align="center" valign="middle"><font face="Arial"> 0.17</font></td><td valign="middle" align="center"><font face="Arial">0.6350</font></td><td align="center" valign="middle"><font face="Arial"> 4.90</font></td><td align="center" valign="middle"><font face="Arial"> 0.10</font></td><td valign="middle" align="center"><font face="Arial"> 0.88</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2xut" target="_blank">2xutA</a></td><td valign="middle"><font face="Arial">
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005215" target="_blank">GO:0005215</a></span>
<span title="oligopeptide transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006857" target="_blank">GO:0006857</a></span>
<span title="peptide transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015197" target="_blank">GO:0015197</a></span>
<span title="peptide transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015833" target="_blank">GO:0015833</a></span>
</font></td></tr>
			<tr><td align="center" valign="middle"><font face="Arial">3</font></td><td align="center" valign="middle"><font face="Arial"> 0.17</font></td><td valign="middle" align="center"><font face="Arial">0.6215</font></td><td align="center" valign="middle"><font face="Arial"> 5.46</font></td><td align="center" valign="middle"><font face="Arial"> 0.08</font></td><td valign="middle" align="center"><font face="Arial"> 0.92</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1pv6" target="_blank">1pv6A</a></td><td valign="middle"><font face="Arial">
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="valine biosynthetic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009099" target="_blank">GO:0009099</a></span>
<span title="acetolactate synthase activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003984" target="_blank">GO:0003984</a></span>
<span title="transketolase activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004802" target="_blank">GO:0004802</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015293" target="_blank">GO:0015293</a></span>
<span title="carbohydrate transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008643" target="_blank">GO:0008643</a></span>
<span title="isoleucine biosynthetic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009097" target="_blank">GO:0009097</a></span>
<span title="acetolactate synthase complex"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005948" target="_blank">GO:0005948</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="plasma membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" target="_blank">GO:0005886</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">4</font></td><td align="center" valign="middle"><font face="Arial"> 0.16</font></td><td valign="middle" align="center"><font face="Arial">0.6347</font></td><td align="center" valign="middle"><font face="Arial"> 5.41</font></td><td align="center" valign="middle"><font face="Arial"> 0.08</font></td><td valign="middle" align="center"><font face="Arial"> 0.93</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1pw4" target="_blank">1pw4A</a></td><td valign="middle"><font face="Arial">
<span title="plasma membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" target="_blank">GO:0005886</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="glycerol metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006071" target="_blank">GO:0006071</a></span>
<span title="transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005215" target="_blank">GO:0005215</a></span>
<span title="glycerol-3-phosphate transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015169" target="_blank">GO:0015169</a></span>
<span title="glycerol-3-phosphate transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015794" target="_blank">GO:0015794</a></span>
<span title="transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055085" target="_blank">GO:0055085</a></span>
</font></td></tr>
			<tr><td align="center" valign="middle"><font face="Arial">5</font></td><td align="center" valign="middle"><font face="Arial"> 0.16</font></td><td valign="middle" align="center"><font face="Arial">0.6114</font></td><td align="center" valign="middle"><font face="Arial"> 5.11</font></td><td align="center" valign="middle"><font face="Arial"> 0.04</font></td><td valign="middle" align="center"><font face="Arial"> 0.86</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2gfp" target="_blank">2gfpA</a></td><td valign="middle"><font face="Arial">
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="drug transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006855" target="_blank">GO:0006855</a></span>
<span title="plasma membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" target="_blank">GO:0005886</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="drug transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015238" target="_blank">GO:0015238</a></span>
<span title="drug transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015893" target="_blank">GO:0015893</a></span>
<span title="transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055085" target="_blank">GO:0055085</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">6</font></td><td align="center" valign="middle"><font face="Arial"> 0.14</font></td><td valign="middle" align="center"><font face="Arial">0.4089</font></td><td align="center" valign="middle"><font face="Arial"> 6.03</font></td><td align="center" valign="middle"><font face="Arial"> 0.09</font></td><td valign="middle" align="center"><font face="Arial"> 0.64</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3cwb" target="_blank">3cwbC</a></td><td valign="middle"><font face="Arial">
<span title="respiratory chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469" target="_blank">GO:0070469</a></span>
<span title="metal ion binding"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872" target="_blank">GO:0046872</a></span>
<span title="mitochondrion"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005739" target="_blank">GO:0005739</a></span>
<span title="mitochondrial inner membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005743" target="_blank">GO:0005743</a></span>
<span title="respiratory electron transport chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022904" target="_blank">GO:0022904</a></span>
<span title="oxidoreductase activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491" target="_blank">GO:0016491</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="electron carrier activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055" target="_blank">GO:0009055</a></span>
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="electron transport chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900" target="_blank">GO:0022900</a></span>
</font></td></tr>
			<tr><td align="center" valign="middle"><font face="Arial">7</font></td><td align="center" valign="middle"><font face="Arial"> 0.13</font></td><td valign="middle" align="center"><font face="Arial">0.4604</font></td><td align="center" valign="middle"><font face="Arial"> 7.10</font></td><td align="center" valign="middle"><font face="Arial"> 0.08</font></td><td valign="middle" align="center"><font face="Arial"> 0.81</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mkt" target="_blank">3mktA</a></td><td valign="middle"><font face="Arial">
<span title="drug transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006855" target="_blank">GO:0006855</a></span>
<span title="drug transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015238" target="_blank">GO:0015238</a></span>
<span title="antiporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015297" target="_blank">GO:0015297</a></span>
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055085" target="_blank">GO:0055085</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">8</font></td><td align="center" valign="middle"><font face="Arial"> 0.12</font></td><td valign="middle" align="center"><font face="Arial">0.3832</font></td><td align="center" valign="middle"><font face="Arial"> 7.37</font></td><td align="center" valign="middle"><font face="Arial"> 0.04</font></td><td valign="middle" align="center"><font face="Arial"> 0.70</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mk7" target="_blank">3mk7A</a></td><td valign="middle"><font face="Arial">
<span title="cytochrome-c oxidase activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004129" target="_blank">GO:0004129</a></span>
<span title="iron ion binding"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506" target="_blank">GO:0005506</a></span>
<span title="electron carrier activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055" target="_blank">GO:0009055</a></span>
<span title="aerobic respiration"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060" target="_blank">GO:0009060</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="heme binding"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037" target="_blank">GO:0020037</a></span>
<span title="plasma membrane respiratory chain complex IV"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0045278" target="_blank">GO:0045278</a></span>
<span title="oxidation-reduction process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114" target="_blank">GO:0055114</a></span>
</font></td></tr>
			<tr><td align="center" valign="middle"><font face="Arial">9</font></td><td align="center" valign="middle"><font face="Arial"> 0.11</font></td><td valign="middle" align="center"><font face="Arial">0.4239</font></td><td align="center" valign="middle"><font face="Arial"> 6.07</font></td><td align="center" valign="middle"><font face="Arial"> 0.07</font></td><td valign="middle" align="center"><font face="Arial"> 0.66</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2fyn" target="_blank">2fynA</a></td><td valign="middle"><font face="Arial">
<span title="plasma membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" target="_blank">GO:0005886</a></span>
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="electron transport chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900" target="_blank">GO:0022900</a></span>
<span title="respiratory chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469" target="_blank">GO:0070469</a></span>
<span title="metal ion binding"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872" target="_blank">GO:0046872</a></span>
<span title="electron carrier activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055" target="_blank">GO:0009055</a></span>
<span title="oxidoreductase activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491" target="_blank">GO:0016491</a></span>
<span title="respiratory electron transport chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022904" target="_blank">GO:0022904</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">10</font></td><td align="center" valign="middle"><font face="Arial"> 0.11</font></td><td valign="middle" align="center"><font face="Arial">0.4120</font></td><td align="center" valign="middle"><font face="Arial"> 5.96</font></td><td align="center" valign="middle"><font face="Arial"> 0.09</font></td><td valign="middle" align="center"><font face="Arial"> 0.64</font></td><td align="left" valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1be3" target="_blank">1be3C</a></td><td valign="middle"><font face="Arial">
<span title="respiratory chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469" target="_blank">GO:0070469</a></span>
<span title="mitochondrial inner membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005743" target="_blank">GO:0005743</a></span>
<span title="mitochondrion"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005739" target="_blank">GO:0005739</a></span>
<span title="electron carrier activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055" target="_blank">GO:0009055</a></span>
<span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span>
<span title="transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810" target="_blank">GO:0006810</a></span>
<span title="membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020" target="_blank">GO:0016020</a></span>
<span title="electron transport chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900" target="_blank">GO:0022900</a></span>
<span title="oxidoreductase activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491" target="_blank">GO:0016491</a></span>
<span title="respiratory electron transport chain"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022904" target="_blank">GO:0022904</a></span>
<span title="metal ion binding"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872" target="_blank">GO:0046872</a></span>
</font></td></tr>
   </tbody></table>
<p>
       <table style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
     <tbody><tr><th colspan="6" align="left"><font face="Arial" size="3"><br>Consensus prediction of GO terms</font></th><th></th><td>&nbsp;</td></tr>
      <tr><td>
      <table cellpadding="2" cellspacing="3" style="font-family:Monospace;font-size:13px;background:#F2F2F2">
       <tbody><tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Molecular Function</b></font></td>
      <td align="center"><font face="Arial"><span title="hexose:hydrogen symporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=term=GO:0009679" target="_blank">GO:0009679</a></span></font></td>
      <td align="center"><font face="Arial"><span title="galactose transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=term=GO:0005354" target="_blank">GO:0005354</a></span></font></td>
      <td align="center"><font face="Arial"><span title="arabinose transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=term=GO:0042900" target="_blank">GO:0042900</a></span></font></td>
      <td align="center"><font face="Arial"><span title="fucose transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=term=GO:0015150" target="_blank">GO:0015150</a></span></font></td>
      <td align="center"><font face="Arial"><span title="transferase activity, transferring aldehyde or ketonic groups"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=term=GO:0016744" target="_blank">GO:0016744</a></span></font></td>
      <td align="center"><font face="Arial"><span title="glycerol transmembrane transporter activity"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=term=GO:0015168" target="_blank">GO:0015168</a></span></font></td>
</tr>
<tr><td><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Biological Process</b></font></td>
      <td align="center"><font face="Arial"><span title="transmembrane transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055085" target="_blank">GO:0055085</a></span></font></td>
      <td align="center"><font face="Arial"><span title="hexose transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008645" target="_blank">GO:0008645</a></span></font></td>
      <td align="center"><font face="Arial"><span title="pentose transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015750" target="_blank">GO:0015750</a></span></font></td>
      <td align="center"><font face="Arial"><span title="hexose metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019318" target="_blank">GO:0019318</a></span></font></td>
      <td align="center"><font face="Arial"><span title="carboxylic acid biosynthetic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046394" target="_blank">GO:0046394</a></span></font></td>
      <td align="center"><font face="Arial"><span title="valine metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006573" target="_blank">GO:0006573</a></span></font></td>
      <td align="center"><font face="Arial"><span title="isoleucine metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006549" target="_blank">GO:0006549</a></span></font></td>
      <td align="center"><font face="Arial"><span title="amine biosynthetic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009309" target="_blank">GO:0009309</a></span></font></td>
      <td align="center"><font face="Arial"><span title="polyol transport"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015791" target="_blank">GO:0015791</a></span></font></td>
      <td align="center"><font face="Arial"><span title="alditol metabolic process"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019400" target="_blank">GO:0019400</a></span></font></td>
</tr>
<tr><td><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.42</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Cellular Component</b></font></td>
      <td align="center"><font face="Arial"><span title="integral to membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021" target="_blank">GO:0016021</a></span></font></td>
      <td align="center"><font face="Arial"><span title="plasma membrane"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886" target="_blank">GO:0005886</a></span></font></td>
      <td align="center"><font face="Arial"><span title="macromolecular complex"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032991" target="_blank">GO:0032991</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cytoplasmic part"><a href="https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0044444" target="_blank">GO:0044444</a></span></font></td>
</tr>
<tr><td><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.60</font></td>
      <td align="center"><font face="Arial">0.52</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
      <td align="center"><font face="Arial">0.33</font></td>
  </tr>
   </tbody></table>
  </td></tr></tbody></table>
</p><p>
       <table style="width:1200px;font-size:12px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tbody><tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is a combined measure for evaluating global and local similarity between query and template protein. It's range is [0-1] and higher values indicate more confident predictions.</td></tr>
      <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
      <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
      <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
      <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
      <tr><td valign="top">(f)</td><td align="justify">The second table shows a consensus GO terms amongst the top scoring templates. The GO-Score associated with each prediction is defined as the average weight of the GO term, where the weights are assigned based on Cscore<sup>GO</sup> of the template.</td></tr>
     </tbody></table>
  </p></div>
</td></tr>
<tr><td><br></td></tr>
<tr><td>
<h3 align="center">[Click on <a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/output/S359535/S359535_results.tar.bz2" download="">S359535_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>
<hr><br>
<div style="position:relative;left:5px;"><table style="font-family:Arial;font-size:14px;"><tbody><tr><td colspan="2">Please cite the following articles when you use the I-TASSER server:</td></tr><tr>
</tr><tr><td valign="top">1.</td><td>Yang Zhang. I-TASSER: Fully automated protein structure prediction in CASP8. Proteins, 77 (Suppl 9): 100-113, 2009.</td></tr>
<tr><td valign="top">2.</td><td>Ambrish Roy, Jianyi Yang, Yang Zhang. COFACTOR: an accurate comparative algorithm for structure-based protein function annotation. Nucleic Acids Research, 40: W471-W477, 2012.</td></tr>
<tr><td valign="top">3.</td><td> Jianyi Yang, Yang Zhang. I-TASSER server: new development for protein structure and function predictions, Nucleic Acids Research, 43: W174-W181, 2015.</td></tr>
</tbody></table></div><br></td></tr>
</tbody></table>

</font><div id="clazzloader-status" style="position: absolute; bottom: 885px; left: 4px; padding: 2px 8px; z-index: 109007; color: yellow; font-family: Arial, sans-serif; font-size: 10pt; white-space: nowrap; opacity: 0.05; display: none; background-color: rgb(142, 0, 0);">Application loaded.</div></body></html>